### PR TITLE
new dependency for windows and modified scripts

### PR DESCRIPTION
**Pull request by mex20**
added a dependency of "mkdirp" for determining if "./data/testrpc/" exists and making it if it does not exist in windows and created a new testrpc-win script in package.json.

### CODE REVIEW

**Comment by mex20** [comment]
It appears that I never actually tried to do a pull request either in the correct way, or at all for this update to have a win-testrpc script, so to make it easier for Windows and *nix machines to be able to run this code with the same features.

File: package.json [hunk 1]
```diff
@@ -27,7 +27,8 @@
     "build": "npm run clean && npm run build-server",
     "test": "npm run eslint && npm run mocha",
     "eslint": "eslint src/. test/. --config .eslintrc.json",
-    "testrpc": "[ ! -d data/testrpc ] && mkdir -p data/testrpc; node node_modules/ethereumjs-testrpc/build/cli.node.js -d --ws --gasLimit 6500000 --db data/testrpc",
+    "testrpc-win": "mkdirp data/testrpc && node node_modules/ethereumjs-testrpc/build/cli.node.js -d --ws --gasLimit 6500000 --db data/testrpc",
```
Comment (perissology) [inline package.json]: Can you remove the `testrpc-win` and just use `mkdirp` in the `testrpc` script. `mkdirp` seems to work with osx & I assume linux.

Comment (mex20) [reply package.json]: Sure thing, I will update it shortly.

**Comment by mex20** [comment]
The requested change has been accomplished.

**Comment by perissology** [comment]
Thanks!

**Pull request merged**